libraries/jevmodal/media/css/jevmodal.css

```diff
@@ -161,7 +162,7 @@
 	max-height:100%;
 }
 .jevmodal .modal-dialog {
-    height: 100%;
+    height: 95%;
```

Comment:
Is this to deal with the scrollbar issue?  I've dealt with this before and thought I had a better solution - from memory its usually a padding or margin issue and using the height CSS won't help on small screens .  Is there a URL where you can see the problem in action?